db/seeds.rb

```diff
@@ -14,7 +14,7 @@
 
 ##############################################################################
 # Default development settings are different from production scenario
-
+FeatureFlag.enable(:profile_admin)
```

Comment:
Question (potentially blocking): This enables the new admin onboarding, but the local development DB will be seeded with users, so the new admin onboarding would never be seen no?

Reply:
@nickytonline I don't for see this causing any issues, since there are no tests associated with the minimal seeded flow. It was just the prep work for https://github.com/forem/forem/pull/14259. I can remove it and leave the file blank and add it to #14259. Thoughts?

Reply:
It makes sense to add it to #14259. I'm not sure what you mean by leave the file blank though, as this is the seed file for local development.

Reply:
Ok so I looked at this commit again and realized I copied this incorrectly before I committed 🤦🏽‍♀️ (hence your comment above and the confusion from myside ). 

It was meant to be 
```
FeatureFlag.enable(:creator_onboarding)
```
and not `:profile_admin`.

In #14259 I will be introducing the Cypress tests that will depend on this `creator_onboarding` feature flag being in place. 

Now that I'm on the same page - I'd actually like to leave the flag in the seed file to prep it for the next PR, because we are renaming files anyways to indicate that we'll be testing the onboarding flow.

Reply:
Ignore whatever I said, this commit needs to be removed 🤦🏽‍♀️ This commit was a mistake and was just a test on the normal `seeds.rb`. I'm clearly very tired and have been working with too many PR's today 😅

Reply:
😅